src/Listeners/TrimFailedJobs.php

```diff
@@ -16,11 +16,11 @@ class TrimFailedJobs
     public $lastTrimmed;
 
     /**
-     * How many hours to wait in between each trim.
+     * How many minutes to wait in between each trim.
      *
      * @var int
      */
-    public $frequency = 23;
+    public $frequency = 5040;
```

Comment:
Why not keep this at hours? Otherwise this is going to be a breaking change I believe

Reply:
I am not sure how this can break things for real. In fact, this initial value is no longer used in the code.